cmd/create/project.go

```diff
@@ -0,0 +1,69 @@
+package create
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/lyft/flyteidl/gen/pb-go/flyteidl/admin"
+	"github.com/lyft/flytestdlib/logger"
+	"gopkg.in/yaml.v2"
+
+	cmdCore "github.com/lyft/flytectl/cmd/core"
+)
+
+//go:generate pflags ProjectConfig
+
+// Config hold configuration for project create flags.
+type ProjectConfig struct {
```

Comment:
@evalsocket I think you can now revive this PR, incorporating the new way introduced by @pmahindrakar-oss